.circleci/config.yml

```diff
@@ -178,10 +178,9 @@ commands:
       - run:
           name: Generate android docs and create PR for review.
           command: |
-            if [[ $CIRCLE_TAG == android-v* ]]; then
+            if [[ $CIRCLE_TAG == android-v* ]] && [[$CIRCLE_TAG != *alpha*]] && [[$CIRCLE_TAG != *beta*]] && [[$CIRCLE_TAG != *rc*]]; then
```

Comment:
does it mean we won't generate API docs for alpha, beta and rc?
we should generated API docs but we should not bump version in android-docs repo.

Reply:
I see, we should config it in script not here.